src/deadline/client/cli/_groups/bundle_group.py

```diff
@@ -92,9 +101,9 @@ def validate_parameters(ctx, param, value):
 )
 @click.option(
     "--job-attachments-file-system",
-    help="The method for accessing files from job attachments. "
-    + "COPIED means to copy files to the host and "
-    + "VIRTUAL means to load files when needed with a virtual file system.",
+    help="The method workers use to access job attachments. "
+    + "COPIED means to copy files to the worker and "
+    + "VIRTUAL means to load files as needed from a virtual file system.",
```

Comment:
We should clarify that VIRTUAL is only available on Linux SMF workers; workers where VIRTUAL is not available will fall-back to COPIED.

Reply:
Nice, I'll add a note about the fallback. I don't think we should specifically call out Linux vs Windows support since that's likely to change in the future.